hwdb.d/meson.build

```diff
@@ -43,37 +45,35 @@ if conf.get('ENABLE_HWDB') == 1
                 command : make_autosuspend_rules_py,
                 capture : true,
                 install : true,
-                install_dir: udevhwdbdir)
+                install_dir : udevhwdbdir,
+        )
 
-        install_data(hwdb_files_notest,
-                     hwdb_files_test,
-                     install_dir : udevhwdbdir)
+        install_data(hwdb_files_notest, hwdb_files_test, install_dir : udevhwdbdir)
 
         if install_sysconfdir
-                meson.add_install_script('sh', '-c',
-                                         mkdir_p.format(sysconfdir / 'udev/hwdb.d'))
+                meson.add_install_script('sh', '-c', mkdir_p.format(sysconfdir / 'udev/hwdb.d'))
 
-                meson.add_install_script('sh', '-c',
-                                         'test -n "$DESTDIR" || @0@/systemd-hwdb update'.format(rootbindir))
+                meson.add_install_script(
+                        'sh',
+                        '-c',
+                        'test -n "$DESTDIR" || @0@/systemd-hwdb update'.format(rootbindir),
+                )
```

Comment:
Those very carefully formatted to use the same pattern: `sh`, `-c` on one line, the command on the other. This reformatting blows up the whitespace and reduces consistency.